MainView.ux

```diff
@@ -3,14 +3,27 @@
 		<Material.Drawer ux:Binding="Drawer" Edge="Left">
 			<Text Alignment="Center">Hello!</Text>
 		</Material.Drawer>
-			
-		<DockPanel>
 
+		<DockPanel>
 			<Material.AppBar Dock="Top">
 				<Material.AppBarIcon Dock="Left" Icon="Menu" Clicked="{openDrawerPanel}" />
 				<Material.AppBarIcon Dock="Right" Icon="DotsVertical" Clicked="{toggleMenu}" />
 				<Material.AppBarIcon Dock="Right" Icon="AccountPlus"  />
 			</Material.AppBar>
+
+			<Rectangle Dock="Top" Margin="5">
+				<StackLayout/>
+				<Stroke Width="1" Color="Black"/>
+				<Material.Text Alignment="Center" Margin="5" Value="Buttons"/>
+				<StackPanel Orientation="Horizontal">
+					<Material.Button Text="Button - Enabled" Raised="true" Enabled="true" BackgroundColor="Material.Blue100" InkColor="Material.Blue300"/>
+					<Material.Button Text="Button - Disabled" Raised="true" Enabled="false" BackgroundColor="Material.Blue100"/>
+				</StackPanel>
+				<StackPanel Orientation="Horizontal">
+					<Material.Button Text="Flat Button - Enabled" Raised="false" Enabled="true" BackgroundColor="#FFFFFF00" InkColor="Material.Blue300"/>
+					<Material.Button Text="Flat Button - Disabled" Raised="false" Enabled="false" BackgroundColor="#FFFFFF00"/>
```

Comment:
Is the background on these fully transparent on purpose? Wouldn't you want to reuse the same color as on the raised buttons, `Material.Blue200`?